cmake/modules/FindGLEW.cmake

```diff
@@ -62,6 +62,12 @@ if (WIN32)
         PATH_SUFFIXES
             Release/${ARCH}
         DOC "The GLEW library")
+
+    if ("${GLEW_LIBRARY}" MATCHES "glew32s(d|)")
+        add_definitions("/DGLEW_STATIC")
```

Comment:
Oh, the unstated bit in my comment is that glew stores GL functions as global function pointer variables of the form __glewApiName -- e.g. glDrawBuffers is internally stored as a global function pointer called __glewDrawBuffers, thus hitting the scenario I described above.